recipe/meta.yaml

```
@@ -1,35 +1,36 @@
+{% set name = "twine" %}
 {% set version = "3.7.1" %}
 
 package:
-  name: twine
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/t/twine/twine-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 28460a3db6b4532bde6a5db6755cf2dce6c5020bada8a641bb2c5c7a9b1f35b8
 
 build:
-  noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [py<36]
+  number: 1
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # depends on readme_renderer not available for linux-s390x
+  skip: true  # [py<36 or s390x]
   entry_points:
     - twine = twine.__main__:main
 
 requirements:
   host:
     - pip
     - python
-    - setuptools >=45
+    - setuptools
     - setuptools_scm >=6.0
     - toml
     - wheel
   run:
+    - python
```

```suggestion
    - python >=3.6
```
For noarch packages it's still relevant to have python >=3.6 only in run